stablehlo/integrations/python/mlir/dialects/stablehlo.py

```diff
@@ -17,3 +17,7 @@
 # pylint: disable=wildcard-import,relative-beyond-top-level,g-import-not-at-top
 from ._stablehlo_ops_gen import *
 from .._mlir_libs._stablehlo import *
+
+
+# Version number for StableHLO Python APIs. Increment on all Python API changes.
+api_version = 1
```

Comment:
As discussed offline, let's turn this into a StablehloModule function that returns a counter which is also incremented on all C++ API changes.